### DISCUSSION TITLE
How to probe rotary axis "A"?

### DISCUSSION THREAD

**Discussion by 7cnc** [terjeio/ioSender#131] [General]
Hello
Is there any way to probe rotary axis "A"?

**Comment by terjeio** [comment]
Currently not from the UI.
[G38.x](http://www.linuxcnc.org/docs/2.6/html/gcode/gcode.html#sec:G38-probe) commands should work but you have to get the position from a `$#` command and manually enter a new (derived?) value in the DRO (or in the offsets tab) afterwards?

**Reply by 7cnc** [reply]
When I use the Gcode G38.x it registers the probe coordinate in some variable?

**Reply by 7cnc** [reply]
When I run the g38.2 command it returns to the console [PRB:-149,000,-154,000,-61,118,62,600:1]. Is there any way to save these coordinates in an external file?

**Reply by 7cnc** [reply]
> Atualmente não é da IU. Comandos
> [G38.x](http://www.linuxcnc.org/docs/2.6/html/gcode/gcode.html#sec:G38-probe) devem funcionar, mas você tem que obter a posição de um `$#`comando e inserir manualmente um novo valor (derivado?) No DRO (ou na guia offsets) depois.

It would be possible to give an example?

**Reply by terjeio** [reply]
> Is there any way to save these coordinates in an external file?

Not without code changes - can you describe your use case?

> It would be possible to give an example?

For how to enter values in the DRO? When you click in a DRO field it becomes editable, after changing the value press  \<Return\> to set it.
Or did you have something else in mind?
